autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -137,10 +153,10 @@ private void scaleUp(int scalingAdjustment, int totalNodes) {
             scalingStatus = ScalingStatus.FAILED;
             statusReason = getMessageForCBException(e);
             LOGGER.error("Couldn't trigger upscaling for host group '{}', cluster '{}', desiredNodeCount '{}', error '{}' ",
-                    hostGroup, cluster.getStackCrn(), desiredNodeCount, statusReason, e);
+                    hostGroup, cluster.getStackCrn(), desiredHostGroupNodeCount, statusReason, e);
             metricService.incrementMetricCounter(MetricType.CLUSTER_UPSCALE_FAILED);
         } finally {
-            processAutoscalingTriggered(scalingAdjustment, totalNodes, statusReason, scalingStatus);
+            processAutoscalingTriggered(scalingAdjustment, hostGroupNodeCount, statusReason, scalingStatus);
```

Comment:
Mentioned elsewhere. The decision to scale up, if affected by max-size, needs to be explicitly logged - to avoid confusion, and questions around "Why did this not scale-up to the configured size"

Reply:
done. logged to our logs and added to console also